data/themes/darktable.css

```diff
@@ -825,16 +841,16 @@ dialog combobox window
 {
   opacity: 1;  /* this is needed for tooltip on/off feature with Shift+t shortcut */
   background-color: @tooltip_bg_color;
-  border-radius: 0.2em;
+  border-radius: 2px;
   border: 0;
   outline-style: none;
   min-height: 0.4em;
   min-width: 0.4em;
-  padding: 0.2em 0;
+  padding: 2px 0;
 }
```

Comment:
Why changing them? Having related sizes is better here and 0.2em works great with your changes. So I suggest to put them back as that don't really change your PR.